chibuezeayogu/Events-Manager
# #159215500 Configure RuboCop

**What does the PR do?**
- This PR adds `Rubocop` rules to ensure that `RoR` style guide is adhered to.

**Description of Task to be completed?**
- Ensures that `RuboCop` is properly configured

**How should this be manually tested?**
- Clone the repository [Events-Manager](https://github.com/chibuezeayogu/Events-Manager.git) and
- Checkout to  this branch `ch-configure-rubocop-159215500`
- run `rubocop` on the terminal
- ensure all `files are inspected and no offenses detected`

**What are the relevant pivotal tracker stories?**
- [#159215500](https://www.pivotaltracker.com/story/show/159215500)

**Screenshots**
- None
